taichi/transforms/alg_simp.cpp

```diff
@@ -164,6 +164,11 @@ class AlgSimp : public BasicStmtVisitor {
     if ((fast_math || is_integral(stmt->ret_type)) &&
         irpass::analysis::same_value(stmt->lhs, stmt->rhs)) {
       // fast_math or integral operands: a / a -> 1
+      if (stmt->lhs->ret_type->is<TensorType>() ||
```

Comment:
Should we simply place a check of `TensorType` at the beginning of this pass?

Reply:
Sure we could do that instead. Just want to place here as a reminder (implement these for TensorType later)